libkbfs/md_ops.go

```diff
@@ -46,12 +46,19 @@ func (md *MDOpsStandard) convertVerifyingKeyError(ctx context.Context,
 }
 
 func (md *MDOpsStandard) verifyWriterKey(
-	ctx context.Context, rmds *RootMetadataSigned) error {
+	ctx context.Context, rmds *RootMetadataSigned, isTlfFinal bool) error {
```

Comment:
Yeah, the metadata block may not be final (pre-reset) but the passed handle may be.